src/types/onyx/Fund.ts

```diff
@@ -0,0 +1,32 @@
+type AdditionalData = {
+    isBillingCard?: boolean;
+    isP2PDebitCard?: boolean;
+};
+
+type AccountData = {
+    additionalData?: AdditionalData;
+    addressName?: string;
+    addressState?: string;
+    addressStreet?: string;
+    addressZip?: number;
```

Comment:
Here too, I feel like most of these aren't optional...

Reply:
I didn't modify these , just moved what was in the old Card.ts file into Fund.ts

Reply:
I get that, but that's not really a good reason to not fix issues with code 😄